capstone-frontend/src/App.test.jsx

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
 import { render } from '@testing-library/react';
-import App from './App';
+import LeftSideBar from './components/LeftSideBar';
+import Navbar from './components/Navbar';
+import RightSideBar from './components/RightSideBar';
+
 
 const container = document.createElement("div");
 document.body.appendChild(container);
 
-test('renders sidebar', () => {
- render(<App />, container);
-  const leftSidebar = document.getElementById("left-sidebar-container");
+const getUserBookLists = () => {
+  return [
+    { id: '1', name: 'Best Books' },
+    { id: '2', name: 'Fantasy' },
+    { id: '3', name: 'Sci-fi' }
+  ];
+};
+
+test('renders left sidebar', () => {
+  render(<Router><LeftSideBar bookLists={getUserBookLists()} /></Router>, container);
```

Comment:
Good job just rendering the minimum you need for each test, and breaking the initial test into these clear, bite-sized units!